README.md

```diff
@@ -18,14 +18,16 @@
 
 # Table Of Contents
 * [Developing](#developing)
+   * [Contribution Guidelines](#contribution-guidelines)
    * [Requirements](#requirements)
    * [Building](#building)
 * [Using Conan with Nexus Repository Manager 3](#using-conan-with-nexus-repository-manager-3)
 * [Installing the plugin](#installing-the-plugin)
+   * [Permanent Renstall](#permanent-reinstall)
```

Comment:
Typo here (Renstall should be Reinstall)

Reply:
Ups, thanks. Fixed 7873f221cbb0b6eff54baaf7a47b171563173e50